modules/core/04-channel/keeper/keeper.go

```diff
@@ -570,3 +570,14 @@ func (k Keeper) iterateHashes(ctx sdk.Context, iterator db.Iterator, cb func(por
 		}
 	}
 }
+
+// hasInflightPackets returns true if there are packet commitments stored at the specified channel and
```

Comment:
I think its small enough to be tested indirectly by the places that call it but can add it to export_test and write a small test for it.